### DISCUSSION TITLE
table 组件实现修改单个单元格属性

### DISCUSSION THREAD

**Discussion by ohhhfish** [layui/layui#1574] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.3

### 问题描述

我想实现双击表格某单元格，使此单元格变成可编辑的状态，在未双击之前是无法编辑的。
我翻了一下表格的文档，没有找到这种功能，请问有办法实现吗

### 业务代码

```auto
-
```

### 浏览器

Chrome 120.0.6099.218（正式版本） （64 位）

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
文档查找 `editTrigger` 属性
ref https://layui.dev/docs/2/table/#options
